migrations/011_alter_runs_sat-fields.up.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE runs
+    ADD COLUMN playbook_name varchar,
+    ADD COLUMN playbook_run_url varchar,
+    ADD COLUMN initiator varchar,
```

Comment:
let's call this "principal" for consistency with the API

Reply:
done!